db/migrate/20230406061717_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.string :bio
+      t.integer :posts_counter
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] There is a duplicate of this table which makes it impossible to run migrate after create the db:

### First Table
![create-user1](https://user-images.githubusercontent.com/26597432/230608136-58171b83-43c3-4090-afe6-f23b184efe82.png)

### Error while trying to migrate
![duplicateMigrate-error](https://user-images.githubusercontent.com/26597432/230608202-a3a300c6-fd97-4db8-b505-8d6533303f30.png)